media-gfx/imagemagick/imagemagick-7.1.1.35.ebuild

```diff
@@ -0,0 +1,247 @@
+		$(use_with X x)
+		$(use_with zip)
+		$(use_with zlib)
+		--without-autotrace
+		$(use_with postscript dps)
+		$(use_with djvu)
+		--with-dejavu-font-dir="${EPREFIX}"/usr/share/fonts/dejavu
+		$(use_with fftw)
+		$(use_with fpx)
+		$(use_with fontconfig)
+		$(use_with truetype freetype)
+		$(use_with postscript gslib)
+		$(use_with graphviz gvc)
+		$(use_with heif heic)
+		$(use_with jbig)
+		$(use_with jpeg)
+		$(use_with jpeg2k openjp2)
+		$(use_with jpegxl jxl)
+		$(use_with lcms)
+		$(use_with lqr)
+		$(use_with lzma)
+		$(use_with openexr)
+		$(use_with pango)
+		$(use_with png)
+		$(use_with raw)
+		$(use_with svg rsvg)
+		$(use_with tiff)
+		$(use_with webp)
+		$(use_with corefonts windows-font-dir "${EPREFIX}"/usr/share/fonts/corefonts)
+		$(use_with wmf)
+		$(use_with xml)
+
+		# Default upstream (as of 6.9.12.96/7.1.1.18 anyway) is open
+		# For now, let's make USE=hardened do 'limited', and have USE=-hardened
+		# reflect the upstream default of 'open'.
+		#
+		# We might change it to 'secure' and 'limited' at some point.
+		# See also bug #716674.
+		--with-security-policy=$(usex hardened limited open)
+	)
+
+	if use perl; then
+		perl_check_env
+		perl_set_version
+		local perl_options=(
+			INSTALLDIRS=vendor
+			INSTALLVENDORARCH="\"${VENDOR_ARCH}\""
+			INSTALLVENDORMAN3DIR="\"${EPREFIX}/usr/share/man/man3\""
+		)
+		myeconfargs+=( --with-perl-options="${perl_options[*]}" )
+	fi
+
+	CONFIG_SHELL="${BROOT}"/bin/bash econf "${myeconfargs[@]}"
+}
+
+src_test() {
+	# Install default (unrestricted) policy in $HOME for test suite, bug #664238
+	local _im_local_config_home="${HOME}/.config/ImageMagick"
+	mkdir -p "${_im_local_config_home}" || \
+		die "Failed to create IM config dir in '${_im_local_config_home}'"
+	cp "${FILESDIR}"/policy.test.xml "${_im_local_config_home}/policy.xml" || \
+		die "Failed to install default blank policy.xml in '${_im_local_config_home}'"
+
+	local im_command= IM_COMMANDS=()
+	if [[ ${PV} == 9999 ]] ; then
+		IM_COMMANDS+=( "magick -version" ) # Show version we are using -- cannot verify because of live ebuild
+	else
+		IM_COMMANDS+=( "magick -version | grep -q -- \"${MY_PV}\"" ) # Verify that we are using version we just built
+	fi
+	IM_COMMANDS+=( "magick -list policy" ) # Verify that policy.xml is used
+	IM_COMMANDS+=( "emake check" ) # Run tests
+
+	for im_command in "${IM_COMMANDS[@]}"; do
+		eval "${S}"/magick.sh \
+			${im_command} || \
+			die "Failed to run \"${im_command}\""
+	done
+}
+
+src_install() {
+	# Ensure documentation installation files and paths with each release!
+	emake \
+		DESTDIR="${D}" \
+		DOCUMENTATION_PATH="${EPREFIX}"/usr/share/doc/${PF}/html \
+		install
+
+	rm -f "${ED}"/usr/share/doc/${PF}/html/LICENSE
+	dodoc AUTHORS.txt README.md
+
+	if use perl; then
+		find "${ED}" -type f -name perllocal.pod -exec rm -f {} +
+		find "${ED}" -depth -mindepth 1 -type d -empty -exec rm -rf {} +
+	fi
+
+	find "${ED}" -name '*.la' -exec sed -i -e "/^dependency_libs/s:=.*:='':" {} +
```

Comment:
not exactly sure what this does, but i think it misses a `|| die`
Also, wouldn't it be better to move this line after the next `find`. (first we `sed` all `*.la` files and then we remove some of them anyway..)

Reply:
_Hello @mm1ke,_

Thank you for reviewing this PR!

I've found that this line was added by **Samuli Suominen** in **2011**:
  - [Reset dependency_libs to '' in libtool files.](https://gitweb.gentoo.org/archive/repo/gentoo-2.git/commit/media-gfx/imagemagick?id=ca863d41)
  - [Drop the useless for loop when we can do this by single find. Thanks to Flameeyes for noticing.](https://gitweb.gentoo.org/archive/repo/gentoo-2.git/commit/media-gfx/imagemagick?id=293bc089)

It remains unchanged since then, only `# libtool is required for loading plugins` comment has gone.

Not sure if we still need this nowadays, but perhaps it's better to ask someone who have expertise in the libtool stuff.

Reply:
@mm1ke I've updated the PR and applied changes that you suggested. Thanks!

Reply:
> @mm1ke I've updated the PR and applied changes that you suggested. Thanks!

Thx, i've also had a look at the build. Unfortunately the sed is still relevant.